File: frontend/src/models/Review.ts
export type Review = {
  reviewId: number;
  result: ReviewResult;
  reviewerId: number;
};

export enum ReviewResult {
  STRONG_ACCEPT = "Strong accept",
  ACCEPT = "Accept",
  BORDERLINE_PAPER = "Borderline paper",
  WEAK_REJECT = "Weak reject",
  REJECT = "Reject",
}

File: backend/src/main/java/com/example/demo/entity/Review.java
package com.example.demo.entity;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity
@Table(name = "REVIEW")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Review {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long reviewId;

    private String result; //strong accept, accept, weak accept, borderline paper, weak reject, reject and strong reject

    @ManyToOne
    @JoinColumn(name = "proposalId")
    private Proposal proposal;

    private boolean isApproved;

}
